test/integration/standard_filter_test.rb

```diff
@@ -852,19 +852,14 @@ def test_all_filters_never_raise_non_liquid_exception
       { 1 => "bar" },
       ["foo", 123, nil, true, false, Drop, ["foo"], { foo: "bar" }],
     ]
-    test_types.each do |first|
-      test_types.each do |other|
-        (@filters.methods - Object.methods).each do |method|
-          arg_count = @filters.method(method).arity
-          arg_count *= -1 if arg_count < 0
-          inputs = [first]
-          inputs << ([other] * (arg_count - 1)) if arg_count > 1
-          begin
-            @filters.send(method, *inputs)
-          rescue Liquid::ArgumentError, Liquid::ZeroDivisionError
-            nil
-          end
-        end
+    (@filters.public_methods - Object.public_methods).each do |method|
```

Comment:
`@filters.public_methods(false)` would return `[]` since `@filters` is an instance of a class that includes the `StandardFilters` module.  However, we could use `StandardFilters.public_instance_methods(false)` to get those methods directly.  I tested `(@filters.public_methods - Object.public_instance_methods) == StandardFilters.public_instance_methods(false)` locally and it returns `true`